mat/triangular.go

```diff
@@ -452,15 +452,39 @@ func (t *TriDense) MulTri(a, b Triangular) {
 		defer restore()
 	}
 
-	// TODO(btracey): Improve the set of fast-paths.
+	// Inspect types here, helps keep the loops later clean(er).
```

Comment:
I'd elide these comments since the code already says what the comments say. But it can be left.

See the Comments section of http://doc.cat-v.org/bell_labs/pikestyle